backend/src/model/dto.rs

```diff
@@ -34,24 +34,41 @@ pub struct ConfigDto {
     pub version: i32,
 }
 
-#[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
+/// Represents seeds of a user.
+#[allow(clippy::missing_docs_in_private_items)]
 #[typeshare]
 #[derive(Serialize, Deserialize, ToSchema)]
 pub struct SeedDto {
+    /// The record id of the seed.
     pub id: i32,
+    /// An additional name for the seed.
     pub name: String,
+    /// The variety of the seed. Currently unused.
     pub variety: Option<String>,
```

Comment:
Leftover, I created #462

```suggestion
    /// The variety of the seed. Should be removed, see #462.
    pub variety: Option<String>,
```